website/docusaurus.config.js

```diff
@@ -347,7 +347,7 @@ const config = {
       },
       docs: {
         sidebar: {
-          autoCollapseCategories: true,
+          autoCollapseCategories: false,
```

Comment:
maybe just drop the full line as it's the default behaviour

Reply:
```suggestion
```